### DISCUSSION TITLE
SOLVED: In OrcaSlicer UI where do I turn skirt on or off?

### DISCUSSION THREAD

**Discussion by DougJoseph** [xblax/flashforge_ad5m_klipper_mod#129] [Q&A] [answered]
Where in OrcaSlicer UI do I turn skirt on or off?

So sorry if I'm overlooking something simple, but I've searched both in the interface and on the web and cannot locate it.

Thanks in advance for any help!

**Comment by cmenard** [comment]
I think you might be looking for the brim settings.  They are in the Others tab under Bed Adhesion.

**Reply by DougJoseph** [reply]
> you might be looking for the brim settings

Thanks, however, brim settings have no impact on this skirt. I have "no brim" selected, and this skirt still happens.

**Reply by cmenard** [reply]
Same place, Skirt loops.  0 should disable it.

**Reply by DougJoseph** [reply]
I have no such option. I'm running OrcaSlicer 2.0.0. I am pretty sure I had no such option in the previous version either. What could make my screen not have something your screen has? Screen shots:

<img width="1728" alt="Screenshot 2024-04-04 at 11 27 43 PM" src="https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/75eb523b-42d8-49d4-b38d-779dddde4fdd">

<img width="1728" alt="Screenshot 2024-04-04 at 11 27 52 PM" src="https://github.com/xblax/flashforge_adm5_klipper_mod/assets/8921397/df17aed8-0722-4a78-a787-896dbc65adbf">

**Comment by DougJoseph** [comment]
Thanks, @cmenard.  I'm currently asking about it in the OrcaSlicer Discord for help.

**Reply by Giterdun2** [reply]
Wow.. that is really strange.. Only thing I can think of is that from your previous videos, you mentioned you were using a MAC.. so may have something to do with THAT version.. This is what I see on Windows 10, with the latest version.. 
![Others](https://github.com/xblax/flashforge_adm5_klipper_mod/assets/165448658/3cb28525-734e-409b-980f-fb25721df6ee)

**Comment by DougJoseph** [comment, marked as answer]
OK, so I was advised of the answer on the OrcaSlicer Discord help channel:

I was in the "Objects" mode instead of "Global" mode...

If I switch over to "Global" mode then I see the Skirt settings...

:-)

**Reply by consp** [reply]
Hadn't even noticed there was a "objects" mode 🤣 good to know (though back to useing prusaslicer again).
